enclave/src/main/java/org/hyperledger/besu/enclave/EnclaveClientException.java

```diff
@@ -14,7 +14,7 @@
  */
 package org.hyperledger.besu.enclave;
 
-public class EnclaveClientException extends IllegalArgumentException {
+public class EnclaveClientException extends EnclaveException {
```

Comment:
What is the difference between a `EnclaveClientException` and a `EnclaveException`, is there value in actually having two, would just one exception suffice?

Reply:
Not certain if one would suffice everywhere. But certainly in the rpcs there is no need to differentiate that it why I introduced a common EnclaveException to avoid adding a multi-catch in all the rpcs.

Reply:
Changed back to just the seperate exceptions without a exception type hierarchy